tests/drivers/uart/uart_mix_fifo_poll/src/main.c

```diff
@@ -28,7 +28,7 @@
 #elif defined(CONFIG_BOARD_ATSAME54_XPRO)
```

Comment:
What if `CONFIG_BOARD_ATSAME54_XPRO` is defined? The files use `DT_CHOSEN`. Will it work? If yes, then it seems those defines are not needed anymore and could be removed to avoid noise.

Same with `CONFIG_BOARD_STM32L562E_DK`

Reply:
af2250a3c8d8585cabdb168ad5b931c0d39a7b6e this commit.

Reply:
You are mixing things up, only uart_basic_api test uses DT_CHOSEN and DEVICE_DT_GET. For uart_mix_fifo_poll and uart_async_api, the CONFIG_UART_CONSOLE_ON_DEV_NAME is replaced by DT_LABEL(DT_CHOSEN(zephyr_console)), therefore the logic is still valid.

Reply:
whopsie